src/MusicStore/Components/SystemClock.cs

```diff
@@ -0,0 +1,22 @@
+﻿using System;
+
+namespace MusicStore.Components
+{
+    /// <summary>
+    /// Provides access to the normal system clock.
+    /// </summary>
+    public class SystemClock : ISystemClock
+    {
+        /// <inheritdoc />
+        public DateTime UtcNow
+        {
+            get
+            {
+                // The clock measures whole seconds only, to have integral expires_in results, and
```

Comment:
This is not necessary here, but miliisecond time resolution should be different depening on SQL server versions, products, or sometimes on its configruation. And Album data should be stored in DB. So, I thought it's ok to keep this code here. But let me know if you want to remove this part, and simply return dattime..